docs/Introduction.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 1
+---
+
+Welcome! 👋
+
+Manta is a set of easy-to-use and secure (open source and auditable code) protocols based on Substrate frameworks. It uses zk-SNARK to provide complete end-to-end anonymity, high throughput, and high cross-chain interoperability (it supports Polkadot and its parachain mainstream cryptocurrencies and stable coins).
+
+## Our Mission
+
+We at Manta Network believe that DeFi, for now, has its most value in permissionless and anti-surveillance. In contrast, current services in DeFi have their deficiency in privacy, scalability, usability, and liquidity. **Under the current landscape, only when a stable coin that fully supports anonymity and cross-chain interoperability and the relevant DeFi protocol being created can the tremendous value of DeFi be released and deployed, and it is at this point that DeFi will have the chance to overtake the traditional financial industry. That's the reason why we build Manta Network.**
+
+## Manta Network Overview
+
+Manta's underlying logic is to mint private coins/tokens with stable coins and other base tokens on the contract at a minting value ratio of 1:1. Apart from that, a decentralized private swap platform can support the exchange of these privacy tokens.
+
+![Manta Overview](/img/manta_overview.png)
+
+The Manta Network mainly consists of two protocols: **the decentralized private payment protocol** and the **decentralized private token exchange protocol.**
+
+## Products
+
+### MantaPay: Privacy Preserving Multi-Asset Decentralized Payment
+
+Is based on zk-SNARK. It allows users to swap Polkadot and its Parachain tokens with corresponding private tokens, pay with privacy tokens, and redeem base coins from private tokens.
+
+### MantaSwap: Privacy Preserving Decentralized Exchange
+
+DAX protocol is based on AMM and zk-SNARK. Users can anonymously trade private versions of tokens on the platform between each other. The price formation method of the decentralized private token exchange protocol is consistent with the mainstream AMM. The revenue model of Manta tokens is based on the rebate and redeem of swap usage fees and transaction fees. In the future, the team will also develop products like synthetic asset stable coins based on Manta.
```

Comment:
```suggestion
MantaSwap is a Decentralized EXchange (DEX) protocol is based on an Automated Market Maker (AMM) and zk-SNARK. The protocol creates a platform where users can anonymously trade private versions of tokens. MantaSwap uses a price formation method that is consistent with mainstream AMM, and _the revenue model of Manta tokens is based on the rebate and redeem of swap usage fees and transaction fees._ 
```

I think this should be DEX not DAX?

I'm also not sure what this sentence means: "The revenue model of Manta tokens is based on the rebate and redeem of swap usage fees and transaction fees", so we should clarify.